_components/in-page-navigation/in-page-navigation.md

```diff
@@ -15,26 +15,8 @@ redirect_from:
 - /in-page-nav/
 - /components/in-page-nav/
 subnav:
-- text: Preview
-  href: '#in-page-navigation-preview'
-- text: Code
-  href: '#in-page-navigation-code'
-- text: Guidance
-  href: '#in-page-navigation-guidance'
-- text: When to use
-  href: '#when-to-use-the-in-page-navigation'
-- text: When to consider something else
-  href: '#when-to-consider-something-else-in-page-navigation'
-- text: Usability guidance
-  href: '#usability-guidance-in-page-navigation'
-- text: Accessibility
-  href: '#accessibility-in-page-navigation'
-- text: Package
-  href: '#in-page-navigation-package'
-- text: References
-  href: '#references'
-- text: Latest updates
-  href: '#changelog'
+  - text: in page navigation accessibility tests
```

Comment:
**Polish:** This navigation item should be capitalized and hyphenated.

```suggestion
  - text: In-page navigation accessibility tests
```

Reply:
Resolved in [5b6b02e](https://github.com/uswds/uswds-site/pull/2863/commits/5b6b02e37396e05112ca7be66889a42238fad443)